HyperPlatform/HyperPlatform.vcxproj

```diff
@@ -122,6 +123,12 @@
       <DisableSpecificWarnings>5040;%(DisableSpecificWarnings)</DisableSpecificWarnings>
     </ClCompile>
     <Link />
+    <Link>
+      <AdditionalDependencies>%(AdditionalDependencies);$(KernelBufferOverflowLib);$(DDK_LIB_PATH)ntoskrnl.lib;$(DDK_LIB_PATH)hal.lib;$(DDK_LIB_PATH)wmilib.lib</AdditionalDependencies>
+    </Link>
+    <Inf>
```

Comment:
Please remove this empty tag for ease of review for the future. (I am aware of that it is auto-generated changes).

Same as line 146.